### PR TITLE
Jasminum sambac knowledge

### PR DESCRIPTION
**Pull request by szemmour-rh**
[#835](https://github.com/instructlab/taxonomy/issues/835)

### CODE REVIEW

File: knowledge/science/botany/jasminum/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by: Samy Zemmouri
```
Comment (jjasghar) [inline knowledge/science/botany/jasminum/qna.yaml:1]: Please put your GitHub Username here.

File: knowledge/science/botany/jasminum/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+domain: flowers
+seed_examples:
+- answer: The English common name of "Arabian jasmine", Jasminum sambac is due to it being widely cultivated in Southwest of Saudi Arabia, Yemen and Oman|
```
Comment (jjasghar) [inline knowledge/science/botany/jasminum/qna.yaml:4]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please resolve these issues then we can run the linter.

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by szemmour-rh** [comment]
requested modification done

**Comment by jjasghar** [comment]
Closed due to the combined PR here: https://github.com/instructlab/taxonomy/pull/1140

**Pull request closed without merging**